plugins/org.jkiss.dbeaver.model.ai/src/org/jkiss/dbeaver/model/ai/metadata/MetadataProcessor.java

```diff
@@ -122,9 +122,10 @@ public DAICompletionMessage createMetadataMessage(
                 "Your query must start with \"SELECT\" and be enclosed with triple backslash on new lines. " +
                 "Talk naturally, as if you were talking to a human.");
         } else if (isChatAPI) {
-            sb.append("Perform SQL completion. Start response with SELECT keyword. Start every line with \"--\".");
+            sb.append("Perform SQL completion. Start response with SELECT keyword.\n" +
+                " Avoid using Markdown. Start every comment line with \"--\".");
         } else {
-            sb.append("Perform SQL completion.");
+            sb.append("Perform SQL completion. Avoid using Markdown");
```

Comment:
Probably lines 124-129 could be just
```java
        } else {
            sb.append("Perform SQL completion. Start your response with \"SELECT\".");
        }
```

Reply:
No, unfortunately Gemini tends to use markdown if not discouraged